app/src/main/java/com/kickstarter/ui/viewholders/MessageViewHolder.java

```diff
@@ -18,66 +16,59 @@
 import com.squareup.picasso.Picasso;
 
 import butterknife.Bind;
-import butterknife.BindDimen;
 import butterknife.ButterKnife;
 
 import static com.kickstarter.libs.rx.transformers.Transformers.observeForUI;
 
 public final class MessageViewHolder extends KSViewHolder {
   private final MessageHolderViewModel.ViewModel viewModel;
 
-  protected @Bind(R.id.message_body_card_view) CardView messageBodyCardView;
```

Comment:
all of the changes, except setting the `deliveryStatusTextView` output, are related to the message view refactoring that correctly shows/hides recipient/sender message views rather than changing the alignment with an output